Northstar.Client/mod/scripts/vscripts/ui/menu_mod_settings.nut

```diff
@@ -1022,6 +1152,37 @@ void function SendTextPanelChanges( var textPanel )
 					Hud_SetText( textPanel, GetConVarString( c.conVar ) )
 				}
 				break
+			case "color":
+				try {
+						table color = StringToColors( GetConVarString(c.conVar) )
+						Hud_SetColor(colorVGUI, color.r, color.g, color.b, color.a)
+					}
+				catch ( ex )
+					{
+						printt(ex)
+						ThrowInvalidValue("This setting is a color, and only accepts a four of numbers - you put something we could not parse!\n\n( Use \".\" for the int like \"255 255 255 255\", not \",\". )")
+						Hud_SetColor(colorVGUI, 0, 0, 0, 0)
+					}
+				break
+			case "alpha":
+				try
+				{
+					SetConVarFloat( c.conVar, newSetting.tofloat() )
+				}
+				catch ( ex )
+				{
+					printt( ex )
+					ThrowInvalidValue( "This setting is a float, and only accepts a number - we could not parse this!\n\n( Use \".\" for the floating point, not \",\". )" )
+				}
+				if ( c.sliderEnabled )
+				{
+					var panel = Hud_GetParent( textPanel )
+					MS_Slider s = file.sliders[ int ( Hud_GetScriptID( panel ) ) ]
+
+					MS_Slider_SetValue( s, GetConVarFloat( c.conVar ) )
+				}
+				Hud_SetAlpha(colorVGUI, int(newSetting * 255))
```

Comment:
Fails compile on this line (`newSetting` is a string)

Reply:
fixed